core/state_processor_test.go

```diff
@@ -302,6 +303,90 @@ func TestStateProcessorErrors(t *testing.T) {
 	}
 }
 
+// TestStateProcessorErrors tests the output from the 'core' errors
+// as defined in core/error.go. This error is generated when the
+// blockchain imports a bad block with a transaction from a
+// non-whitelisted address.
+func TestBadWhitelistedBlock(t *testing.T) {
```

Comment:
Nit: can we update the test name to: TestBadTxAllowListBlock and update the comment as well?

Reply:
fixed